syntax = "proto3";

message MessageHeader{
    bytes service_name=1;
    bytes method_name=2;
    int32 body_size=3;
}